<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TPClocheClass</class>
 <widget class="QMainWindow" name="TPClocheClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TPCloche</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cloche3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloche4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloche1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloche2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 2</string>
    </property>
   </widget>
   <widget class="QListWidget" name="log">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 1 : 1(Pavé num)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 2 : 2(Pavé num)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 3 : 3(Pavé num)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 4 : 4(Pavé num)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Clavier :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TPCloche.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cloche1</sender>
   <signal>clicked()</signal>
   <receiver>TPClocheClass</receiver>
   <slot>cloche1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cloche2</sender>
   <signal>clicked()</signal>
   <receiver>TPClocheClass</receiver>
   <slot>cloche2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cloche3</sender>
   <signal>clicked()</signal>
   <receiver>TPClocheClass</receiver>
   <slot>cloche3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cloche4</sender>
   <signal>clicked()</signal>
   <receiver>TPClocheClass</receiver>
   <slot>cloche4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cloche1()</slot>
  <slot>cloche2()</slot>
  <slot>cloche3()</slot>
  <slot>cloche4()</slot>
 </slots>
</ui>
